nuget/src/it/test-nuget-common-flow.groovy

```diff
@@ -32,15 +32,15 @@ runCommand(targetPath, String.format(
     configPath))
 
 output = runCommand(targetPath, String.format(
-    "$nugetExec list Org.Carlspring -ConfigFile %s",
-    configPath))
+    "$nugetExec list Org.Carlspring -Source %s",
+    storageUrl))
 assert output.contains("Org.Carlspring.Strongbox.Examples.Nuget.Mono")
 
 output = runCommand(targetPath, String.format(
     "$nugetExec delete Org.Carlspring.Strongbox.Examples.Nuget.Mono 1.0.0 -Source %s -NonInteractive -ConfigFile %s",
     storageUrl,
     configPath))
 output = runCommand(targetPath, String.format(
-    "$nugetExec list Org.Carlspring -ConfigFile %s",
-    configPath))
+    "$nugetExec list Org.Carlspring -Source %s",
+    storageUrl))
```

Comment:
This should probably be a separate command?

Reply:
Changed to separate command